addOns/pscanrules/pscanrules.gradle.kts

```diff
@@ -6,15 +6,32 @@ description = "The release quality Passive Scanner rules"
 zapAddOn {
     addOnName.set("Passive scanner rules")
     addOnStatus.set(AddOnStatus.RELEASE)
-    zapVersion.set("2.7.0")
+    zapVersion.set("2.8.0")
 
     manifest {
         author.set("ZAP Dev Team")
+        extensions {
+            register("org.zaproxy.zap.extension.pscanrulesBeta.payloader.ExtensionPayloader") {
+                classnames {
+                    allowed.set(listOf("org.zaproxy.zap.extension.pscanrules.payloader"))
+                }
+                dependencies {
+                    addOns {
+                        register("custompayloads") {
+                            version.set("0.9.*")
+                        }
+                    }
+                }
+            }
+        }
     }
 }
 
 dependencies {
     implementation("com.shapesecurity:salvation:2.7.0")
+    compileOnly(parent!!.childProjects.get("custompayloads")!!)
+
+    testImplementation(parent!!.childProjects.get("custompayloads")!!)
 
```

Comment:
Could be removed.

Reply:
I’m pretty sure this was copied from another addon that’s currently (or was till this change) using Custom Payloads. Did you mean both entries or just the second?

Reply:
I'm referring to this empty line (to be together with existing test implementations).

Reply:
Ok